### DISCUSSION TITLE
How to use loop (while True) in asyncTeleBot ?

### DISCUSSION THREAD

**Discussion by VietnameseRick** [eternnoir/pyTelegramBotAPI#1507] [Q&A]
Hi, can anyone show me any example of using while True loop in asyncTeleBot ? I want the Loop the startup automatically after the user setup is completed and no need for any command. Thanks for your help ^^

**Comment by coder2020official** [comment]
Why do you need while True? I did not understand the purpose :/

**Reply by VietnameseRick** [reply]
Sorry for late response, I want the bot to always call the API of wallet/portfolio crypto price tracking 24/7

**Comment by Badiboy** [comment]
Start it in separate thread.

**Comment by coder2020official** [comment]
Create an asynchronous task and run
